docs/UserGuide.md

```diff
@@ -3,436 +3,713 @@ layout: page
+
+### Example Command:
+```
+add n/ <NAME> p/ <PHONE> e/ <EMAIL> a/ <ADDRESS> j/ <JOBNAME> i/ <INCOME> [t/ <TIER>] [rn/ <REMARK>]
+```
+- **Mandatory Components**: Flags such as `n/`, `p/`, `e/`, `a/`, `j/`, and `i/` must be followed by valid arguments like the name, phone number, and job title.
+- **Optional Components**: Flags like `t/` and `rn/` are enclosed in square brackets, indicating they are optional.
+
+## 5.2 Data Modification Commands
+
+### 5.2.1 Adding a new client {#add-command}
+
+**Purpose:** Save detailed records of a new client. 
+
+Each client's record includes their name, contact number, email, occupation, and income. You can also enter the optional fields for credit card tier and remark here. Otherwise, new users are assigned a default value of "N.A".
+
+**Command Format:**
   ```
-  delete <INDEX>
+  add n/ <NAME> p/ <PHONE> e/ <EMAIL> a/ <ADDRESS> j/ <JOBNAME> i/ <INCOME> [t/ <TIER>] [rn/ <REMARK>]
   ```
-- **Example:**
+* Mandatory Fields: `n/`, `p/`, `e/`, `a/`, `j/`, `i/`
+* Optional Fields: `t/`, `rn/`
+
+For detailed explanations of each flag and acceptable arguments, refer to Sections [4.3 Flags](#43-flags) and [4.4 Arguments](#44-arguments)
+
+**Examples:**
+- Add new customer (without optional fields):
+  ```
+  add n/ JOHN DOE p/ 99007766 e/ mrdoe@ntu.sg a/ com3 j/ doctor i/ 99999
   ```
-  delete 69
+- Add new customer with tier and remark:
   ```
+  add n/ JOHN DOE p/ 99007766 e/ mrdoe@ntu.sg a/ com3 j/ doctor i/ 99999 t/ gold rn/ got anger issue
+  ```
+
+#### What to Expect
+- **On Success:** 
+  - Message: 
+    ```
+    New client added: Name: <NAME>, Phone: <PHONE>, Email: <EMAIL>, Address: <ADDRESS>, Job: <JOB>, Income: <INCOME>, Tier: <TIER>, Remark: <REMARK>.
+    ```
+  - If "Tier" and "Remark" are not provided, they will be set to "N.A." and displayed as such in the success message.
 
-#### Parameters
+- **On Error**
+  - Message: 
+    ```
+    Please verify that your input is in the correct format. Include the following details: n/ <NAME> p/ <PHONE> e/ <EMAIL> a/ <ADDRESS> j/ <JOBNAME> i/ <INCOME> [t/ <TIER>] [rn/ <REMARK>].
+    ```
 
-| Parameter | Expected Format               | Explanation                                                                                                                                 |
-|-----------|-------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------|
-| INDEX     | Integer (1 to the last INDEX) | The INDEX must be a valid integer within the registered range (either the original list or any filtered list after using `filter` command). |
+> **Note on Duplicates:**
+>
+> AgentAssist will prevent duplicate entries if a client with the **same name, email, job, and income** is already saved.  
```

Comment:
This has been amended, refer to #108. But equality is now stricter, with same name, email and phone being the conditions for equality.

Reply:
Appreciate the catch, amended!